packages/recommend-js/src/frequentlyBoughtTogether.tsx

```diff
@@ -26,7 +26,7 @@ const UncontrolledFrequentlyBoughtTogether = createFrequentlyBoughtTogetherCompo
   }
 );
 
-function useFrequentlyBoughtTogether<TObject>(
+export function useFrequentlyBoughtTogether<TObject>(
```

Comment:
Just bringing awareness that this will now be exported from the package, which isn't part of the contract. (Could be fine as long as we don't document it, so we can remove it after the experimental package is removed.)